src/ol/layer/VectorTile.js

```diff
@@ -93,14 +89,19 @@ class VectorTileLayer extends BaseVectorLayer {
 
     super(/** @type {import("./BaseVector.js").Options} */ (baseOptions));
 
-    const renderMode = options.renderMode || VectorTileRenderType.HYBRID;
+    let renderMode = options.renderMode || VectorTileRenderType.HYBRID;
+    if (renderMode === VectorTileRenderType.IMAGE) {
+      //FIXME deprecated - remove this check in v7.
+      //eslint-disable-next-line
+      console.warn('renderMode: "image" is deprecated. Option ignored.')
+      renderMode = undefined;
```

Comment:
Possibly this should be `VectorTileRenderType.HYBRID` ref #12162 ?
I can provide a PR if wanted

Reply:
Thanks for spotting this. #12162 should be an appropriate fix.